README.md

```diff
@@ -21,71 +20,91 @@ Panther is a security analytics platform built for cloud-focused security teams.
 
 Panther enables teams to define detections as code and programmatically upload them to your Panther deployment.
 
-This repository contains all of Panther's built-in detections  installed by default.
+This repository contains all the detections developed by Panther for use in your Panther deployment.
```

Comment:
@k-bailey  is this any better? I agree that this read weird before. but maybe it still does

Reply:
I checked in with Sally S and she confirmed that this updated version reads ok. I think this is ready to ship